app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -99,8 +100,7 @@ def consumer_show
 
     #api :GET, "/owners/:organization_id/environments", N_("List environments for RHSM")
     def rhsm_index
-      organization = find_organization
-      @all_environments = get_content_view_environments(params[:name], organization).collect do |env|
+      @all_environments = get_content_view_environments(params[:name], Organization.current).collect do |env|
```

Comment:
let's use Organization.current which is now properly set by `set_taxonomy` from core